### ISSUE TITLE
ZipFile.createSplitZipFile always calls setSplitLength

### ISSUE THREAD

**Issue by osterwiese** [srikanth-lingala/zip4j#99]
Hi, 

createSplitZipFile always calls setSplitLength. This causes some troble because I do not wand to split the archive (splitArchive = false).

In createSplitZipFileFromFolder the split length is only set if splitArchive = true.
`
 if (splitArchive) {
      zipModel.setSplitLength(splitLength);
 }
`

Can you fix that please?
I'm really happy that the development started again for this project .
Thanks and regards, Peter

**Comment by srikanth-lingala** [comment]
Hi Peter,

I am wondering why you want to use `createSplitZipFile()` when you do not want to split the archive? Why not instead use any one of `addFiles()` methods? One of the differences between 1.x and 2.x of zip4j worth mentioning in this context is that, in 1.x `addFiles()` only adds files to a zip if the zip file exists, but in 2.x, `addFiles()` creates a zip file if it does not exist, and adds files to it. So it is not a prerequisite for a zip file to exist when using `addFiles()` method in 2.x.

It is also interesting for me to know what problem you get even if you use `createSplitZipFile` and set splitArchive to false? A tip here: if you set `splitArchive` to false, you have to pass in the `splitLength` as -1. 

> I'm really happy that the development started again for this project .

Thank you. I am also really glad that I found time to bring this project to life again after several years. Considering the popularity this project was getting, it would have been sad to not support it, so I made time for it in the busy parental life :)

Cheers
Srikanth

**Comment by osterwiese** [comment]
Hi Srikanth,
thanks for your fast reply and for the information.
I was not aware about the change in addFiles(). It works perfect!
When I used createSplitZipFile with splitArchive =false I tried first with 0 (because with createSplitZipFileFromFolder it does not matter). After an exception I tried the expected split minimum and got the next exception. With -1 it works fine, but (to be honest) the diefferent behaviour is a bit missleading. 😃 

Thanks again, you did a great job!

Regards, Peter
